### ISSUE TITLE
Instructions for Pop not clear

### ISSUE THREAD

**Issue by datacubeR** [qtile/qtile#4323]
### The issue:

I can't even make Qtile work, because of errors. I'm using PopOS.
I work as Data Scientist so I use conda to manage my Python packages. I installed qtile and xcffib. As per instructions those are the only needed packages, but when trying to run qtile using qtile start I get the following error message: 

```bash
Traceback (most recent call last):
  File "/home/datacuber/miniconda3/lib/python3.9/site-packages/libqtile/pangocffi.py", line 51, in <module>
    from libqtile._ffi_pango import ffi
  File "/home/datacuber/miniconda3/lib/python3.9/site-packages/libqtile/_ffi_pango.py", line 3, in <module>
    from cairocffi._generated.ffi import ffi as _ffi0
ModuleNotFoundError: No module named 'cairocffi._generated'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/datacuber/miniconda3/bin/qtile", line 5, in <module>
    from libqtile.scripts.main import main
  File "/home/datacuber/miniconda3/lib/python3.9/site-packages/libqtile/scripts/main.py", line 6, in <module>
    from libqtile.scripts import check, cmd_obj, migrate, run_cmd, shell, start, top
  File "/home/datacuber/miniconda3/lib/python3.9/site-packages/libqtile/scripts/check.py", line 27, in <module>
    from libqtile import confreader
  File "/home/datacuber/miniconda3/lib/python3.9/site-packages/libqtile/confreader.py", line 31, in <module>
    from libqtile.backend.x11 import core
  File "/home/datacuber/miniconda3/lib/python3.9/site-packages/libqtile/backend/x11/core.py", line 37, in <module>
    from libqtile.backend import base
  File "/home/datacuber/miniconda3/lib/python3.9/site-packages/libqtile/backend/base.py", line 11, in <module>
    from libqtile import drawer, pangocffi, utils
  File "/home/datacuber/miniconda3/lib/python3.9/site-packages/libqtile/drawer.py", line 35, in <module>
    from libqtile import pangocffi, utils
  File "/home/datacuber/miniconda3/lib/python3.9/site-packages/libqtile/pangocffi.py", line 53, in <module>
    raise ImportError("No module named libqtile._ffi_pango, be sure to run `./scripts/ffibuild`")
ImportError: No module named libqtile._ffi_pango, be sure to run `./scripts/ffibuild`
```

I'm quite familiar with Python, and this is clearly a dependency error with pangocffi. But is this a pip installable package or what? Because it is not in PyPI, and it seems we need to install it using apt or something. 

Can you guide me through the installation process? I'd prefer not install things outside conda environments unless is needed not to mess up my python interpreters. But I'm able to get the Qtile Window manager option at initialization but when starting I get this which is clearly not the default Qtile (and actually looks like Ubuntu):

![image](https://github.com/qtile/qtile/assets/48638337/d395a7e9-94c4-493e-9736-4bb87782e8e4)

I have tried looking at the paths shown in there but it seems that pangocffi.py doesn't exist in my system. 

I think Qtile is very promising, but probably is not well documented for systems other than Arch (I think 90+% of tutorials are using Arch).

Any help will be deeply appreciated. 

Thanks in advance,

Alfonso

### Required:

- [X] I have searched past issues to see if this bug has already been reported.

**Comment by ervinpopescu** [comment]
> I'm quite familiar with Python, and this is clearly a dependency error with pangocffi. But is this a pip installable package or what? Because it is not in PyPI, and it seems we need to install it using apt or something.

You probably need to install latest version of [cairocffi](https://pypi.org/project/cairocffi/) (1.6.0) and [xcffib](https://pypi.org/project/xcffib/) as specified in the [CHANGELOG](https://raw.githubusercontent.com/qtile/qtile/master/CHANGELOG). Can you run these and give me the output?
```bash
qtile --version
pip list | grep 'cairocffi\|xcffib'
```

**Comment by ramnes** [comment]
> I installed qtile and xcffib. As per instructions those are the only needed packages

These are the only Python packages needed, but there are some system dependencies as well. What instructions did you follow? And did you install PangoCairo before installing Qtile? From the docs:

<img width="697" alt="image" src="https://github.com/qtile/qtile/assets/835072/4e7198a0-5972-4bfc-85b7-b49f86560e3e">

**Comment by datacubeR** [comment]
> > I installed qtile and xcffib. As per instructions those are the only needed packages
> 
> These are the only Python packages needed, but there are some system dependencies as well. What instructions did you follow? And did you install PangoCairo before installing Qtile? From the docs:
> 
> <img alt="image" width="697" src="https://user-images.githubusercontent.com/835072/250921855-4e7198a0-5972-4bfc-85b7-b49f86560e3e.png">
> https://qtile.readthedocs.io/en/v0.9.1/manual/install/index.html

I tried this too and it seems it is already installed: 
![image](https://github.com/qtile/qtile/assets/48638337/23c35b1c-2c15-4516-a984-eaee469fdd7b)

**Comment by ramnes** [comment]
I think you either have to install Qtile master branch from git (until we drop a new release) or rollback cairocffi and xcffib to older versions

**Comment by elParaguayo** [comment]
@datacubeR Is this fixed now? Can we close this?

**Comment by datacubeR** [comment]
@elParaguayo I think I didn´t understand would be my solution. I'd prefer not install from master branch (if any other options exists), so in that case, is there any cairocffi and xcffib version that should work for my case?

I would really like to make qtile work, so any help would be awesome... 

Thanks,

Alfonso

**Comment by ramnes** [comment]
Should be fixed with the latest release (0.23.0). :)

Feel free to comment if needed.
